CRM/Core/DAO.php

```diff
@@ -1956,7 +1956,7 @@ public static function composeQuery($query, $params = [], $abort = TRUE) {
     $tr = [];
     foreach ($params as $key => $item) {
       if (is_numeric($key)) {
-        if (CRM_Utils_Type::validate($item[0], $item[1]) !== NULL) {
+        if (CRM_Utils_Type::validate($item[0], $item[1], TRUE, $item[2] ?? 'One of the parameters ') !== NULL) {
```

Comment:
I think the original grammar mistake is grandfathered in - you can't fix that _now_. (grin)

Reply:
It's a sneaky way to tell which errors came via execute query  - like a special code or secret handshake